packages/@react-stately/dnd/src/useDraggableCollectionState.ts

```diff
@@ -101,10 +102,15 @@ export function useDraggableCollectionState(props: DraggableCollectionOptions):
       }
     },
     endDrag(event) {
+      let {
+        isInternalDrop
+      } = event;
+
       if (typeof onDragEnd === 'function') {
         onDragEnd({
           ...event,
-          keys: draggingKeys.current
+          keys: draggingKeys.current,
+          isInternalDrop
```

Comment:
Debatable if this `isInternalDrop` is necessary any more now that `onItemRemove` is no longer a thing. I felt it was still nice to have so that users could recreate the `isInternalDrop` logic in their on drag end without having to track if it is an internal drop themselves